proto-docs/container.md

```diff
@@ -94,20 +99,35 @@ response immediately. After a new block is issued in sidechain, request is
 verified by Inner Ring nodes. After one more block in sidechain, container
 is added into smart contract storage.
 
+Statuses:
+- **OK** (0, [SECTION_SUCCESS](status.md#neo.fs.v2.status.Success)):
+request to remove the container has been sent to the sidechain;
+- [Common failures](status.md#neo.fs.v2.status.CommonFail).
+
 | Name | Input | Output |
 | ---- | ----- | ------ |
 | Delete | [DeleteRequest](#neo.fs.v2.container.DeleteRequest) | [DeleteResponse](#neo.fs.v2.container.DeleteResponse) |
 #### Method Get
 
 Returns container structure from `Container` smart contract storage.
 
+Statuses:
+- **OK** (0, [SECTION_SUCCESS](status.md#neo.fs.v2.status.Success)):
+container read successfully;
```

Comment:
'was' is omitted here and in lines 128, 155. It's ok but in all other lines passive voice is used in its full form, so I'd love to see it uniform.

Reply:
Made the same